1kgenomes_examples/dashboard.md

```diff
@@ -0,0 +1,65 @@
+
+# 1000 Genomes examples
+
+## Introduction
+
+This workspace collects several example notebooks that use the 1000 Genomes public dataset.
+
+To run the examples, **first Duplicate this workspace**.  You can do this via the 'three-dot' menu
+next to the 'Share' button at the top right in the Workbench UI.
+
+Then, in your new workspace, create a JupyterLab R-based cloud environment.  You can take the defaults for
+machine size and number of CPUs. No GPUs are necessary.
+
+Click the link to open the JupyterLab server once the cloud environment is up and running. In the
+left-hand file navigator, click in to `repos/terra-axon-examples/1kgenones_examples`.  In this
+directory you'll see three example notebook files, described below.
+
+When you run the R examples (`R_1k_genomes.ipynb` and `1kg_analysis.ipynb` ), ensure that you select
+the R kernel. For `GWAS_experiments.ipynb`, use the Python 3 kernel.
```

Comment:
Could we provide a hyperlink to instructions for switching from one kernel to another?

Reply:
Good idea, I will make a note to find something for the next iteration.  Now that I fixed the nb to automatically use the R kernel it's not quite so urgent.